Dockerfile-dynamodb

```diff
@@ -0,0 +1,12 @@
+FROM python:rc-buster
+RUN apt-get update && apt-get install -y default-jre && \
```

Comment:
It probably doesn't matter for the prototype, but do you think we pin the JRE and Dynamo DB versions once we're in Beta?

Reply:
Yeah, we definitely need to lock them down at some point. Like you say, it's fine for now.